src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -64,6 +67,7 @@ define(function (require, exports, module) {
         this._color = tinycolor(color);
         this._originalColor = color;
         this._redoColor = null;
+        this._isUpperCase = PreferencesManager.get("uppercaseColors");
```

Comment:
`tinycolor.toString()` actually works as the author intended. It does not ignore the original case, but rather normalizes (to lowercase) it before returning it. There was actually no way to get the original input (it wasn't even stored) until I suggested it and sent a patch in the form of `tinycolor.getOriginalInput()`.